### PR TITLE
Add a short extra lesson on removing files/datasets 

### PR DESCRIPTION
**Pull request by adswa**
This adds a brief and stand-alone episode on removing file contents and/or datasets.

### CODE REVIEW

File: _episodes/92-filesystem-operations.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+> ## Dropping multiple versions of a file
+> If a file has been updated and saved multiple times, the revision history of the dataset will have each version in the annex, and only the most recent versions' content would be dropped.
+>Consider the following example:
+> {% raw %}
+>       datalad run -m "Make a first version" --output local-file-multiple-revision.txt "echo 'This is the first revision' > {outputs}"
+>       # overwrite the contents of the file 
+>       datalad run -m "Make a second version" --output local-file-multiple-revision.txt "echo 'This is the second revision' > {outputs}"
+> {% endraw %}
+>
+>``datalad drop --reckless availability local-file-multiple-revision.txt`` would drop the _current_ revision of the file, but the first version (with the content "``This is the first revision``") would still exist in the datasets history.
```
Comment (mslw) [inline _episodes/92-filesystem-operations.md]:
```suggestion
>``datalad drop --reckless availability local-file-multiple-revision.txt`` would drop the _current_ revision of the file, but the first version (with the content "``This is the first revision``") would still exist in the dataset's history.
```

File: _episodes/92-filesystem-operations.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+{: .language-bash}
+
+~~~
+drop(ok): A.Shashua-Introduction_to_Machine_Learning.pdf (file)
+~~~
+{: .output}
+
+### ``remove`` the easy way: datasets with remote availability
+
+To demonstrate removing a datasets, let's remove the `machinelearning-books` dataset.
```
Comment (mslw) [inline _episodes/92-filesystem-operations.md]:
```suggestion
To demonstrate removing datasets, let's remove the `machinelearning-books` dataset.
```

File: _episodes/92-filesystem-operations.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+  web: https://arxiv.org/pdf/0904.3664v1.pdf
+ok
+~~~
+{: .output}
+
+This output shows that the file contents of this book exist locally and "on the web", more precisely under the URL [arxiv.org/pdf/0904.3664v1.pdf](https://arxiv.org/pdf/0904.3664v1.pdf).
+
+A ``datalad drop <file>`` performs an internal check if the to-be-dropped file content could be re-obtained before dropping it.
+When the file content exists in more than your local system, _and_ at least one of those other location is successfully probed to have this content, dropping its file content succeeds.
```
Comment (mslw) [inline _episodes/92-filesystem-operations.md]: more places/locations than?

File: _episodes/92-filesystem-operations.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+~~~
+# add write permissions to the directory
+chmod -R +w <dataset>
+# remove everything that is left
+rm -rf <dataset>
+~~~
+{: .language-bash}
+## Further reading
+
+This overview only covers the surface of all possible file system operations, and not even all ways in which files or datasets can be removed. For a more complete overview, checkout [handbook.datalad.org/basics/101-136-filesystem.html](http://handbook.datalad.org/en/latest/basics/101-136-filesystem.html) - the navigation bar on the left of the page lists a number of useful file system operations.
```
Comment (mslw) [inline _episodes/92-filesystem-operations.md]:
```suggestion
This overview only covers the surface of all possible file system operations, and not even all ways in which files or datasets can be removed. For a more complete overview, check out [handbook.datalad.org/basics/101-136-filesystem.html](http://handbook.datalad.org/en/latest/basics/101-136-filesystem.html) - the navigation bar on the left of the page lists a number of useful file system operations.
```

**Review by mslw — COMMENTED** [review]
This is great! I left just a few editing comments.

File: _episodes/92-filesystem-operations.md (hunk 1 above)
Comment (adswa) [reply _episodes/92-filesystem-operations.md]: thanks for catching this!

**Comment by adswa** [comment]
thx much for catching all of those!

File: _episodes/92-filesystem-operations.md (hunk 3 above)
Comment (adswa) [reply _episodes/92-filesystem-operations.md]: oops, missed this one, thanks!

Comment (adswa) [reply _episodes/92-filesystem-operations.md]: pushed a fix

**Pull request merged**